bubblewrap.c

```diff
@@ -1327,6 +1333,10 @@ main (int    argc,
   if (!is_privileged)
     opt_unshare_user = TRUE;
 
+  if (opt_unshare_user_try &&
+      stat ("/proc/self/ns/user", &sbuf) == 0)
```

Comment:
Exactly what does the RHEL7 patch stop?

For the record, this is the debian patch:
https://sources.debian.net/src/linux/4.5.4-1/debian/patches/debian/add-sysctl-to-disallow-unprivileged-CLONE_NEWUSER-by-default.patch/

With it, root or CAP_SYSADMIN (i.e. setuid bwrap) is allowed to create new user namespaces, but not regular users, they will fail clone() and unshare() with EPERM.